tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -80,6 +80,30 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 			stdOut = helper.CmdShouldFail("odo", "service", "create")
 			Expect(stdOut).To(ContainSubstring("odo doesn't support interactive mode for creating Operator backed service"))
 		})
+
+		It("should successfully push a second service after a first service is deployed", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs")
+			operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
+			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]-clusterwide`).FindString(operators)
+
+			// create a first service
+			stdOut := helper.CmdShouldPass("odo", "service", "create", fmt.Sprintf("%s/EtcdCluster", etcdOperator), "myetcd1", "--project", commonVar.Project)
+			Expect(stdOut).To(ContainSubstring("Successfully added service to the configuration"))
+
+			// deploy it
+			helper.CmdShouldPass("odo", "push")
+			stdOut = helper.CmdShouldPass("odo", "service", "list")
+			Expect(stdOut).To(ContainSubstring("EtcdCluster/myetcd1"))
+
+			// create a second service
+			stdOut = helper.CmdShouldPass("odo", "service", "create", fmt.Sprintf("%s/EtcdCluster", etcdOperator), "myetcd2", "--project", commonVar.Project)
+			Expect(stdOut).To(ContainSubstring("Successfully added service to the configuration"))
+
+			// deploy it
+			helper.CmdShouldPass("odo", "push")
+			stdOut = helper.CmdShouldPass("odo", "service", "list")
+			Expect(stdOut).To(ContainSubstring("EtcdCluster/myetcd2"))
```

Comment:
Might be a good idea to check for the presence of both the services

Reply:
done, thanks